Dockerfile

```
@@ -0,0 +1,13 @@
+ARG PHP_VERSION=7.4
+FROM php:${PHP_VERSION}-cli
+
+WORKDIR /usr/src/app/
+
+COPY composer-install.sh /usr/src/app/
+
+RUN apt-get update
+RUN apt-get install wget -y
+RUN apt-get install zip -y
+RUN apt-get install git -y
+RUN apt autoremove -y
+RUN ./composer-install.sh
```

```suggestion
RUN set -e \
    && apt-get update \
    && apt-get install -y wget zip git \
    && apt autoremove -y \
    && ./composer-install.sh
```

With every new `RUN` command the image grows as Docker creates a new aufs layer.

---

And yes, 🐳 it is ugly.